htdocs/compta/facture/card.php

```diff
@@ -1790,8 +1790,8 @@
 											$lines[$i]->pa_ht,
 											$label,
 											$array_options,
-											$lines[$i]->situation_percent,
-											$lines[$i]->fk_prev_id,
+											$lines[$i]->situation_percent ?? '',
```

Comment:
Ok for the ??
But if not defined, you should use the "default" value expected by the function instead of ''.
So 100 for situation_percent and 0 for fk_prev_id